app/src/main/res/values-te-rIN/strings.xml

```diff
@@ -0,0 +1,264 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="action_about_us">మా గురించి చర్య</string>
+    <string name="dialog_cancel_sign_up">డైలాగ్ సైన్అప్ రద్దు</string>
+    <string name="menu_item_about">మెను ఐటెమ్ గురించి</string>
+    <string name="menu_item_settings">మెను ఐటమ్ సెట్టింగులు</string>
+    <string name="no_skill_name">సామర్థ్యం గల పేరు లేదు</string>
+    <string name="no_skill_description">ఏ ప్రతిభ వివరణ లేదు</string>
+    <string name="skill_unrated">సామర్థ్యం ధృవీకరించబడలేదు</string>
+    <string name="skill_not_found">ఏ టాలెంట్ లేదు</string>
+    <string name="action_select_query_language">భాషను ఎంచుకోండి</string>
+    <string name="action_select_tts_language">TTS భాషను ఎంచుకోవడానికి దశ</string>
+    <string name="query_language">ప్రశ్న భాష</string>
+    <string name="settings_devices">పరికరాల కోసం సెట్టింగ్లు</string>
+    <string name="cancel">రద్దు</string>
+    <string name="check_email">ఇమెయిల్ చూడండి</string>
+    <string name="reset_password_text">పాస్వర్డ్ టెక్స్ట్ని రీసెట్ చేయండి</string>
+    <string name="invalid_credentials_title">తప్పు ఆధారాల శీర్షిక</string>
+    <string name="invalid_credentials">చెల్లని పత్రం టెస్టిమోనియల్స్</string>
+    <string name="forgot_pass_activity">నేను పాస్వర్డ్ చర్యను మర్చిపోయాను</string>
+    <string name="no">తోబుట్టువుల</string>
```

Comment:
I think you are using google translate directly.
"తోబుట్టువుల" means "siblings"